ogb/lsc/mag240m.py

```diff
@@ -53,6 +54,25 @@ def download(self):
                 print('Stop download.')
                 exit(-1)
 
+    def to_pyg_hetero_data(self):
+        data = HeteroData()
+        path = osp.join(self.dir, 'processed', 'paper', 'node_feat.npy')
+        # Current is not in-memory
```

Comment:
Do you mean:
```
```suggestion
        # Currently in-memory only
```

Reply:
`data["paper"].x = torch.from_numpy(np.load(path, mmap_mode='r'))` is from `@property def paper_label(self)...`, which is called when `self.in_memory` is `False`. So I comment here, to remind myself to enable `in_memory` part.